unittests/test_mapper.py

```diff
@@ -9,16 +9,15 @@
 from bomf.model import Bo4eTyp
 
 
-class _NotImplementedBo4eDataSetMixin:
+class _NotImplementedBo4eDataSetMixin(BaseModel):
     """
     a mixin to inherit from if you'd like to have correct types but don't care about the logic
     """
 
 
-@attrs.define(kw_only=True, auto_attribs=True)
 class _MaLoAndMeLo(_NotImplementedBo4eDataSetMixin):
```

Comment:
```suggestion
class _MaLoAndMeLo(BaseModel, _NotImplementedBo4eDataSetMixin):
```
hier gehört das basemodel hin, statt in das mixin